device/bgp/bgpgracefulrestart.yaml

```diff
@@ -39,7 +39,7 @@ components:
           x-field-uid: 3
         stale_time:
           description: |-
-            This time (in seconds) specifies how long stale information (for the AFI/SAFI) 
+            Duration (in seconds) specifies how long stale information (for the AFI/SAFI) 
```

Comment:
specified -> specifying . Or Duration (in seconds) for which long stale information ....